### PR TITLE
Feature/ganesh

### PR DESCRIPTION
**Pull request by SekarGaneshAppDev**

### CODE REVIEW

File: src/views/grids/ReportForDay.jsx [hunk 1]
```diff
@@ -24,42 +24,52 @@ class ReportForDay extends Component {
...
-            <Link
-              to={
-                {
-                  //   pathname: 'viewvehicle/' + row['_original'].licensePlate
-                }
-              }
-            >
-              {row['_original'].licensePlate}
-            </Link>
+            <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/grids/ReportForDay.jsx:27]: minor : pls avoid inline css

File: src/views/grids/ReportForDay.jsx [hunk 2]
```diff
@@ -24,42 +24,52 @@ class ReportForDay extends Component {
...
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
       },
       {
         Header: 'Date',
         accessor: 'requestServiceDate',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: row => (
+          <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/grids/ReportForDay.jsx:69]: minor : pls avoid inline css

File: src/views/service/ListService.jsx [hunk 3]
```diff
@@ -35,22 +38,54 @@ class ListService extends Component {
...
+          </div>
+        )
       },
       {
         Header: 'Description',
         accessor: 'description',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: ({ row }) => (
+          <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/service/ListService.jsx:63]: minor: inline css

File: src/views/service/ListService.jsx [hunk 4]
```diff
@@ -35,22 +38,54 @@ class ListService extends Component {
       {
         Header: 'Service Name',
         accessor: 'serviceName',
-        className: 'text-center'
+        className: 'text-center',
+        width: 350,
+        Cell: ({ row }) => (
+          <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/service/ListService.jsx:44]: minor: inline css

File: src/views/grids/ReportForWeek.jsx [hunk 5]
```diff
@@ -23,22 +23,28 @@ class ReportForWeek extends Component {
...
-            <Link
-              to={
-                {
-                  //   pathname: 'viewvehicle/' + row['_original'].licensePlate
-                }
-              }
-            >
-              {row['_original'].licensePlate}
-            </Link>
+            <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/grids/ReportForWeek.jsx:26]: minor: inline css

File: src/views/service/ListService.jsx (hunk 4 above)
Comment (varunsapp) [inline src/views/service/ListService.jsx:44]: minor: inline css

File: src/views/service/ListService.jsx [hunk 6]
```diff
@@ -155,7 +190,7 @@ class ListService extends Component {
                       <h3 className="mb-0">List Of Available Services</h3>
                     </Col>
                     <Col sm>
-                      <FormGroup style={{ float: 'right' }}>
+                      <FormGroup style={{ float: 'left', width: '60%' }}>
```
Comment (varunsapp) [inline src/views/service/ListService.jsx:193]: minor: inline css

File: src/views/service/ListService.jsx [hunk 7]
```diff
@@ -173,8 +208,7 @@ class ListService extends Component {
                       <span
                         style={{
                           float: 'right',
```
Comment (varunsapp) [inline src/views/service/ListService.jsx:210]: minor: inline css

File: src/views/service/ListService.jsx [hunk 8]
```diff
@@ -187,7 +221,7 @@ class ListService extends Component {
                         </Button>
                         <CSVLink
                           data={this.state.dataToDownload}
-                          filename={'Service List' + '.csv'}
+                          filename={downFileName + '.csv'}
```
Comment (varunsapp) [inline src/views/service/ListService.jsx:224]: minor code smell: 
add .csv above itself

File: src/views/serviceReport/ServiceReport.jsx [hunk 9]
```diff
@@ -49,62 +50,83 @@ class ServiceReport extends Component {
...
-            <Link
-              to={
-                {
-                  //   pathname: 'viewvehicle/' + row['_original'].licensePlate
-                }
-              }
-            >
-              {row['_original'].licensePlate}
-            </Link>
+            <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:53]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 10]
```diff
@@ -49,62 +50,83 @@ class ServiceReport extends Component {
...
+            </div>
           );
         }
       },
       {
         Header: 'Make',
         accessor: 'make',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:70]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 11]
```diff
@@ -49,62 +50,83 @@ class ServiceReport extends Component {
...
       },
       {
         Header: 'Model',
         accessor: 'model',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:76]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 12]
```diff
@@ -49,62 +50,83 @@ class ServiceReport extends Component {
...
       },
       {
         Header: 'Status',
         accessor: 'status',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:88]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 13]
```diff
@@ -49,62 +50,83 @@ class ServiceReport extends Component {
...
       },
       {
         Header: 'Plan Type',
         accessor: 'planType',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: ({ row }) => (
+          <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:95]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 14]
```diff
@@ -49,62 +50,83 @@ class ServiceReport extends Component {
...
+            </UncontrolledTooltip>
+          </div>
+        )
       },
       {
         Header: 'Due Amount',
         accessor: 'due',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'right' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:113]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 15]
```diff
@@ -49,62 +50,83 @@ class ServiceReport extends Component {
...
       },
       {
         Header: 'Paid Amount',
         accessor: 'paid',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'right' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:119]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 16]
```diff
@@ -49,62 +50,83 @@ class ServiceReport extends Component {
...
         Header: 'Service Date',
         accessor: 'requestedServiceDate',
         className: 'text-center',
-        Cell: ({ row }) => {
-          return moment(row['_original'].requestedServiceDate).format(
-            'DD/MM/YYYY'
-          );
-        }
+        Cell: row => (
+          <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:126]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 17]
```diff
@@ -237,61 +259,34 @@ class ServiceReport extends Component {
...
-                        size="sm"
-                        id="down_pdf"
-                        onClick={this.downloadPdf}
-                      >
-                        <i className="fas fa-file-download"></i> PDF
-                      </Button>
-                      <UncontrolledTooltip placement="top" target={'down_pdf'}>
-                        Download as PDF
-                      </UncontrolledTooltip>
+                      <FormGroup style={{ float: 'right' }}>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:266]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 18]
```diff
@@ -237,61 +259,34 @@ class ServiceReport extends Component {
...
-                        selected={endDate}
-                        onChange={this.edateChange}
-                        dateFormat="yyyy-MM-dd"
-                        placeholderText="Select End Date"
-                      />
+                    <Col md="3">
+                      <FormGroup>
+                        <DatePicker
+                          selected={endDate}
+                          onChange={this.edateChange}
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:281]: minor code smell: ->>this.endDateChange

File: src/views/serviceReport/ServiceReport.jsx [hunk 19]
```diff
@@ -237,61 +259,34 @@ class ServiceReport extends Component {
...
+                          dateFormat="yyyy-MM-dd"
+                          placeholderText="Select End Date"
+                          className="form-control"
+                          width="100%"
+                        />
+                      </FormGroup>
                     </Col>
-                    <Col md="2">
+                    <Col md="2" style={{ marginLeft: '-5rem' }}>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:289]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 20]
```diff
@@ -320,6 +315,38 @@ class ServiceReport extends Component {
                         Reset
                       </UncontrolledTooltip>
                     </Col>
+
+                    <Col md="2" style={{ marginLeft: '5rem' }}>
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:319]: minor: inline css

File: src/views/serviceReport/ServiceReport.jsx [hunk 21]
```diff
@@ -320,6 +315,38 @@ class ServiceReport extends Component {
...
+                        color="primary"
+                        size="sm"
+                        onClick={this.download}
+                        id="down_csv"
+                      >
+                        <i className="fas fa-file-download"></i> CSV
+                      </Button>
+                      <CSVLink
+                        data={dataToDownload}
+                        filename={downFileName + '.csv'}
```
Comment (varunsapp) [inline src/views/serviceReport/ServiceReport.jsx:330]: minor code smell: create filename above itself

File: src/views/users/Users.jsx [hunk 22]
```diff
@@ -89,27 +90,48 @@ class Users extends React.Component {
       {
         Header: 'Name',
         accessor: 'name',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/users/Users.jsx:94]: minor: inline css

File: src/views/users/Users.jsx [hunk 23]
```diff
@@ -89,27 +90,48 @@ class Users extends React.Component {
...
       },
       {
         Header: 'Email',
         accessor: 'email',
-        className: 'text-center'
+        className: 'text-center',
+        width: 250,
+        Cell: ({ row }) => (
+          <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/users/Users.jsx:102]: minor: inline css

File: src/views/users/Users.jsx [hunk 24]
```diff
@@ -89,27 +90,48 @@ class Users extends React.Component {
...
+          </div>
+        )
       },
       {
         Header: 'Phone Number',
         accessor: 'phoneNumber',
-        className: 'text-center'
+        className: 'text-center',
+        width: 150,
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/users/Users.jsx:121]: minor: inline css

File: src/views/users/Users.jsx [hunk 25]
```diff
@@ -89,27 +90,48 @@ class Users extends React.Component {
...
+        className: 'text-center',
+        width: 180
       },
       {
         Header: 'Source Reg',
         accessor: 'sourceofReg',
-        className: 'text-center'
+        className: 'text-center',
+        width: 150,
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/users/Users.jsx:134]: minor: inline css

File: src/views/users/Users.jsx [hunk 26]
```diff
@@ -198,21 +221,43 @@ class Users extends React.Component {
...
           return (
-            <Link
-              to={{
-                pathname:
-                  'vehicle-service-details/' + row['_original'].vehicleId
-              }}
-            >
-              {row['_original'].licensePlate}
-            </Link>
+            <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/users/Users.jsx:224]: minor: inline css

File: src/views/users/Users.jsx [hunk 27]
```diff
@@ -198,21 +221,43 @@ class Users extends React.Component {
...
+        Header: 'Year',
+        accessor: 'year',
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
+      },
+      {
+        Header: 'Color',
+        accessor: 'color',
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/users/Users.jsx:259]: minor: inline css

File: src/views/users/Users.jsx [hunk 28]
```diff
@@ -413,28 +460,14 @@ class Users extends React.Component {
...
-    //     swal.fire({ text: text, type: type });
-    // }
+  componentDidUpdate(newProps) {
+    const { updateUserResponse, deleteUserResponse } = this.props;
+    if (newProps.updateUserResponse !== updateUserResponse) {
+      swal.fire(getAlertToast('success', updateUserResponse.data));
+    }
+    if (newProps.deleteUserResponse !== deleteUserResponse) {
+      swal.fire(getAlertToast('success', deleteUserResponse.data));
+    }
```
Comment (varunsapp) [inline src/views/users/Users.jsx:470]: if api is successful you can fire toasts from actions itself, this logic is not recommended

File: src/views/users/Users.jsx [hunk 29]
```diff
@@ -490,7 +523,10 @@ class Users extends React.Component {
                   SubComponent={() => {
                     return (
                       <div style={{ padding: '20px' }}>
```
Comment (varunsapp) [inline src/views/users/Users.jsx:525]: minor: inline css

File: src/views/vehicle/ListVehicle.jsx [hunk 30]
```diff
@@ -50,42 +52,61 @@ class ListVehicle extends Component {
...
-            <Link
-              to={
-                {
-                  //   pathname: 'viewvehicle/' + row['_original'].licensePlate
-                }
-              }
-            >
-              {row['_original'].licensePlate}
-            </Link>
+            <div style={{ textAlign: 'left' }}>
```
Comment (varunsapp) [inline src/views/vehicle/ListVehicle.jsx:55]: minor: inline css

File: src/views/vehicle/ListVehicle.jsx [hunk 31]
```diff
@@ -50,42 +52,61 @@ class ListVehicle extends Component {
...
+            </div>
           );
         }
       },
       {
         Header: 'Make',
         accessor: 'make',
-        className: 'text-center'
+        className: 'text-center',
+        Cell: row => <div style={{ textAlign: 'left' }}>{row.value}</div>
```
Comment (varunsapp) [inline src/views/vehicle/ListVehicle.jsx:72]: minor: inline css

**Review by varunsapp — APPROVED** [review]

File: src/views/grids/ReportForDay.jsx (hunk 1 above)
Comment (SekarGaneshAppDev) [reply src/views/grids/ReportForDay.jsx:27]: Fixed

File: src/views/service/ListService.jsx (hunk 3 above)
Comment (SekarGaneshAppDev) [reply src/views/service/ListService.jsx:63]: Fixed

File: src/views/service/ListService.jsx (hunk 4 above)
Comment (SekarGaneshAppDev) [reply src/views/service/ListService.jsx:44]: Fixed

File: src/views/grids/ReportForWeek.jsx (hunk 5 above)
Comment (SekarGaneshAppDev) [reply src/views/grids/ReportForWeek.jsx:26]: fixed

File: src/views/users/Users.jsx (hunk 27 above)
Comment (SekarGaneshAppDev) [reply src/views/users/Users.jsx:259]: Fixed

File: src/views/users/Users.jsx (hunk 29 above)
Comment (SekarGaneshAppDev) [reply src/views/users/Users.jsx:525]: Fixed

File: src/views/vehicle/ListVehicle.jsx (hunk 30 above)
Comment (SekarGaneshAppDev) [reply src/views/vehicle/ListVehicle.jsx:55]: Fixed

File: src/views/vehicle/ListVehicle.jsx (hunk 31 above)
Comment (SekarGaneshAppDev) [reply src/views/vehicle/ListVehicle.jsx:72]: Fixed

**Pull request merged**